dDatabase/CDClientDatabase/CDClientTables/CDPackageComponentTable.cpp

```diff
@@ -19,12 +19,11 @@ void CDPackageComponentTable::LoadValuesFromDatabase() {
 	// Now get the data
 	auto tableData = CDClientDatabase::ExecuteQuery("SELECT * FROM PackageComponent");
 	while (!tableData.eof()) {
-		CDPackageComponent entry;
+		auto &entry = entries.emplace_back();
```

Comment:
format: we use left aligned ref and ptr.

Reply:
My spacebar evidently is a C purist 😅